index.js

```diff
@@ -39,51 +55,70 @@ function checkGuess() {
     guessInput.disabled = true;
   }
 
-  if (guess !== targetNumber) {
-    if (guess < targetNumber) {
+  if(guess !== targetNumber && guess > 0 && guess < 100) {
+    if(guess < targetNumber) {
       tooLowMessage.style.display = '';
     } else {
-      tooLowMessage.style.display = '';
+      tooHighMessage.style.display = '';
     }
 
     const remainingAttempts = maxNumberOfAttempts - attempts;
 
     numberOfGuessesMessage.style.display = '';
+    
+    if(attempts === 4) {
+      numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guess remaining`;
+    } else {
     numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+    }
   }
 
-  if (attempts ==== maxNumberOfAttempts) {
+  if(attempts === maxNumberOfAttempts) {
     submitButton.disabled = true;
     guessInput.disabled = true;
+    resetButton.style.display = '';
   }
 
   guessInput.value = '';
-
-  resetButton.style.display = '';
+  // resetButton.style.display = '';
+
+  if(guess < 0) { 
+    negativeMesssage.style.display = '';
+    resetButton.style.display = 'none';
+  } else if(guess > 100) { 
+    belowOneHundredMesssage.style.display = '';
+    resetButton.style.display = 'none';
+  } else if(guess === 0) {
+    zeroMesssage.style.display = '';
+    resetButton.style.display = 'none';
```

Comment:
```suggestion
  if(guess < 1) { 
    negativeMesssage.style.display = '';
    resetButton.style.display = 'none';
  } else if(guess > 100) { 
    belowOneHundredMesssage.style.display = '';
    resetButton.style.display = 'none';
  }
```
I don't think that zero needs a special case.